docs/getting_started/quick_start/load_files.rst

```diff
@@ -0,0 +1,45 @@
+Load files
+========================
+
+
+mol2 files
+------------------------
+
+Create an mbuild.Compound (i.e., the "pentane" variable) by loading a molecule from a `mol2 <http://chemyang.ccnu.edu.cn/ccb/server/AIMMS/mol2.pdf>`_ file.
+
+Import the required mbuild packages.
+
+.. code:: ipython3
+
+    import mbuild as mb
+
+
+Load the "pentane.mol2" file from its directory.
+
+.. code:: ipython3
+
+    pentane = mb.load("path_to_mol2_file/pentane.mol2")
+
+
+CIF files
+------------------------
+
+Build an mbuild.Compound (i.e., the "ETV_triclinic" variable) by loading a `Crystallographic Information File (CIF) <https://www.iucr.org/resources/cif>`_ file and selecting the number of cell units to use in the x, y, and z-dimensions.
+
+
+Import the required mbuild packages.
+
+.. code:: ipython3
+
+    import mbuild as mb
+    from mbuild.lattice import load_cif
+
+
+
+The `CIF <https://www.iucr.org/resources/cif>`_ file is loaded with the load_cif_command. Next, three (3) cell units shall be built for all the x, y, and z-dimensions with the populate function.  Finally, the `CIF <https://www.iucr.org/resources/cif>`_'s residues as 'ETV'.
+
+.. code:: ipython3
+
+    lattice_cif_ETV_triclinic = load_cif("path_to_cif_file/ETV_triclinic.cif")
+    ETV_triclinic = lattice_cif_ETV_triclinic.populate(x=3, y=3, z=3)
+    ETV_triclinic.name = 'ETV'
```

Comment:
Maybe mention at the end of this file other file formats we support loading into mbuild i.e. hoomd_snapshot, .gsd, smarts, parmed etc.

Reply:
This was added to the end of the file

Other file types
------------------------
mBuild also supports loading data or files via hoomd_snapshot, gsd, smarts strings, and parmed structures.